code/__DEFINES/sound.dm

```diff
@@ -34,7 +38,7 @@
 //THIS SHOULD ALWAYS BE THE LOWEST ONE!
 //KEEP IT UPDATED
 
-#define CHANNEL_HIGHEST_AVAILABLE 1008 //CIT CHANGE - COMPENSATES FOR VORESOUND CHANNELS
+#define CHANNEL_HIGHEST_AVAILABLE 1006 //CIT CHANGE - COMPENSATES FOR VORESOUND CHANNELS
```

Comment:
Seemingly, the lower is now 1007

Reply:
Says its need to be the lowest one, reactor is set to 1007 as per #define CHANNEL_REACTOR_ALERT 1007 // reactor sounds so wouldn't it need to go down to 1006?

Reply:
This might be because of splurt's special sounds, i think the channel used for lewd sounds is 1007, on cit it is clearly 1008 because of the vore sounds

Reply:
Lets just get this straight, is it fine for the two to be the same, if so than it can be 1007.